### PR TITLE
Add react example code to the README

### PR DESCRIPTION
**Pull request by IanPhilips**
Submitting this is a talking point, feel free to suggest another way of adding this documentation.

I had to load using the CDN because of #181.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,49 @@ That's it. Easy as apple pie.
     </script>
   </body>
 </html>
+```
+## React example (just loading videomail)
```
Comment (binarykitchen) [inline README.md:76]: Gatsby example

File: README.md [hunk 2]
```diff
@@ -72,6 +72,49 @@ That's it. Easy as apple pie.
...
+```jsx harmony
+import React, { Component } from 'react';
```
Comment (binarykitchen) [inline README.md:78]: no semicolons, use the standard linter

File: README.md [hunk 3]
```diff
@@ -72,6 +72,49 @@ That's it. Easy as apple pie.
...
+import { Helmet } from 'react-helmet';
+
+export default class VideomailReactExample extends Component{
+  state = {
+      videomailClient:null,
+  };
+
+  componentDidMount() {
+    (async() => {
+      console.log("waiting for videomail client library to load via CDN");
```
Comment (binarykitchen) [inline README.md:88]: remove or make it a comment instead

File: README.md [hunk 4]
```diff
@@ -72,6 +72,49 @@ That's it. Easy as apple pie.
...
+      while(window.VideomailClient === undefined)
```
Comment (binarykitchen) [inline README.md:89]: what if the remote resource is not available due to some network issues? this would loop forever ...

File: README.md [hunk 5]
```diff
@@ -72,6 +72,49 @@ That's it. Easy as apple pie.
...
+      })
+    })();
+  };
+  render(){
+    return(
+      <div>
+        <Helmet>
+          <script src="https://cdn.rawgit.com/binarykitchen/videomail-client/2.14.2/prototype/js/videomail-client.min.js" />
+        </Helmet>
+      {
```
Comment (binarykitchen) [inline README.md:107]: { this.state.videomailClient && this.renderVideomail() }

would look easier to read

**Comment by binarykitchen** [comment]
Thanks for this!

Honestly I am not a fan of using CDN. Understand it's a limitation due to Gatsby being a SSR. That would mean your example is actually for Gatsby itself, not React. Because in a normal client-side React app we wouldn't have this problem.

Submitted a review.

**Comment by IanPhilips** [comment]
Good point! Thanks for the review, I'll fix those issues. 

This is definitely a Gatsby example. I can cook up a react example, would you prefer to have the gatsby example on file somewhere or just a plain react example?

**Comment by binarykitchen** [comment]
Thanks. Plain React please.

**Pull request closed without merging**